java/flight/flight-sql-jdbc-driver/src/main/java/org/apache/arrow/driver/jdbc/ArrowFlightPreparedStatement.java

```diff
@@ -39,12 +39,12 @@ public class ArrowFlightPreparedStatement extends AvaticaPreparedStatement
 
   private final ArrowFlightSqlClientHandler.PreparedStatement preparedStatement;
 
-  private ArrowFlightPreparedStatement(final ArrowFlightConnection connection,
-                                       final ArrowFlightSqlClientHandler.PreparedStatement preparedStatement,
-                                       final StatementHandle handle,
-                                       final Signature signature, final int resultSetType,
-                                       final int resultSetConcurrency,
-                                       final int resultSetHoldability)
+  public ArrowFlightPreparedStatement(final ArrowFlightConnection connection,
```

Comment:
Instead of a public constructor, would it be possible to add a new static (package-private) factory method?

Reply:
Changed.